src/widgets/refinement-list/refinement-list.js

```diff
@@ -66,7 +66,7 @@ const renderer = ({
       showMore={showMoreConfig !== null}
       templateProps={renderState.templateProps}
       toggleRefinement={refine}
-      searchFacetValues={searchForFacetValues && searchForItems}
+      searchFacetValues={searchForFacetValues ? searchForItems : () => {}}
```

Comment:
Because `RefinementList` is awaiting a `PropTypes.function` for `searchFacetValues` should we change it to `oneOfType([ ... ])` ?